app/src/main/java/com/mobile/tandil/kotlinbaseproject/listener/ListenerSetParkingSpaces.kt

```diff
@@ -0,0 +1,7 @@
+package com.mobile.tandil.kotlinbaseproject.listener
+
+import java.io.Serializable
+
+interface ListenerSetParkingSpaces : Serializable {
```

Comment:
?

Reply:
why you are using this " : Serializable " ??

Reply:
because in the companion object of the Fragment when it's set the bundle, it use the .setSerializable to do it. So, its necessary to use the : Serializable